launch/scenario_perception_pipeline_benchmark.launch.py

```diff
@@ -0,0 +1,180 @@
+import os
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution
+from launch.conditions import IfCondition
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+from ament_index_python.packages import get_package_share_directory
+from moveit_configs_utils import MoveItConfigsBuilder
+from launch.actions import (
+    DeclareLaunchArgument,
+    ExecuteProcess,
+    LogInfo,
+    RegisterEventHandler,
+)
+from launch.event_handlers import OnProcessExit
+
+
+def generate_launch_description():
+    db_arg = DeclareLaunchArgument(
```

Comment:
Do you need the db_arg?

Reply:
Removed in https://github.com/CihatAltiparmak/moveit_middleware_benchmark/pull/5/commits/caa54960dd6febab1c82f7b63b93a38616339cda commit